composer.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
maybe add

```
    "config": {
       "optimize-autoloader": true,
       "classmap-authoritative": true
     },
```

For more secure autoloader? That way it will only look trough it lists of known files (so the created list of the dumpautoloader command). Which means no loading magic and accidentially loading a wrong file.